stan/math/prim/mat/fun/divide_columns.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_DIVIDE_COLUMNS_HPP
+#define STAN_MATH_PRIM_MAT_FUN_DIVIDE_COLUMNS_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <stan/math/prim/scal/fun/divide.hpp>
+#include <stan/math/prim/scal/meta/scalar_type.hpp>
+#include <vector>
+
+namespace stan {
+namespace math {
+
+/**
+ * Takes Stan data type vector[n] x[D] and divides each
+ * dimension by the specified scalar.
+ *
+ *
+ * @tparam T_x type of elements contained in vector x, usually Eigen::Matrix
+ * @tparam T_s type of element of scalar, usually double or var
+ * @tparam R   number of rows in the submatrix of x
+ * @tparam C   number of columns in the submatrix of x
+ *
+ * @param x std::vector of elements representing a matrix
+ * @param scalar a scalar
+ *
+ */
+template <typename T_x, typename T_s, int R, int C>
+inline typename std::vector<Eigen::Matrix<T_x, R, C>> divide_columns(
+    const std::vector<Eigen::Matrix<T_x, R, C>> &x, const T_s &scalar) {
+  size_t N = x.size();
+  std::vector<Eigen::Matrix<T_x, R, C>> out(N);
+  for (size_t n = 0; n < N; ++n) {
+    out[n] = divide(x[n], scalar);
+  }
+  return out;
+}
+
+/**
+ * Takes Stan data type vector[n] x[D] and divides each
+ * dimension sequentially by each element of the vector
+ *
+ * @tparam T_x type of elements contained in vector x, usually Eigen::Matrix
+ * @tparam T_v vector of elements
+ * @tparam R   number of rows in the submatrix of x
+ * @tparam C   number of columns in the submatrix of x
+ *
+ * @param x std::vector of elements representing a matrix
```

Comment:
Please change this comment. The elements are all matrices, not that the collection of the vector represents a single matrix in some way.